src/dataregistry/registrar_util.py

```diff
@@ -80,3 +94,58 @@ def get_directory_info(path):
                 num_files += subdir_num_files
                 total_size += subdir_total_size
     return num_files, total_size
+
+def calculate_special(name, v_string, v_suffix, dataset_table, engine):
```

Comment:
call this function something like `version_bump`?

Reply:
Yeah, maybe, but I prefer `bump_version` (start with a verb).